pkgs/applications/networking/instant-messengers/bitlbee-facebook/default.nix

```diff
@@ -12,6 +12,10 @@ stdenv.mkDerivation rec {
     sha256 = "1yjhjhk3jzjip13lq009vlg84lm2lzwhac5jy0aq3vkcz6rp94rc";
   };
 
+  # TODO: This patch should be included with the next release after v1.2.1
+  #       these lines and the patch should be removed when this happens.
+  patches = [ ./0001-Bump-the-FB_ORCA_AGENT-version-once-again-208.patch ];
```

Comment:
Can you use fetchpatch to fetch the two commits instead?

Reply:
Done. It's only a single commit, fyi. This doesn't include the base commit in the patch. Though I suppose it may not matter because `patch` probably ignores that anyway. Thought it might be human-friendlier though.